### ISSUE TITLE
Abnormal circulating glutaric acid

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6012]
**Preferred term label:** Abnormal circulating glutaric acid

**Synonyms**

**Definition (free text, please give PubMed ID)** Any deviation form the normal concentration of a glutaric acid in the blood circulation.

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating alpha-ketoglutarate concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of alpha-ketoglutarate in the blood is above the normal range., PMID: 30181211

**Parent term (use hpo.jax.org/app)** Abnormal circulating glutaric acid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating 3-methylglutaric acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 3-methylglutaric acid in the blood is above the normal range., PMID: 31730811

**Parent term (use hpo.jax.org/app)** Abnormal circulating glutaric acid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating 3-methylglutaconic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 3-methylglutaconic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating glutaric acid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0003530, I will add as a synonym

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4703346/

**Comment by pnrobinson** [comment]
added
